common/eth2/src/types.rs

```diff
@@ -1080,6 +1080,8 @@ pub enum EventKind<E: EthSpec> {
     #[cfg(feature = "lighthouse")]
     BlockReward(BlockReward),
     PayloadAttributes(VersionedSsePayloadAttributes),
+    ProposerSlashing(Box<ProposerSlashing>),
+    AttesterSlashing(Box<AttesterSlashing<T>>),
```

Comment:
This will now need to be: 
```suggestion
    AttesterSlashing(Box<AttesterSlashing<E>>),
```